src/_scss/pages/bulkDownload/archive/_archiveForm.scss

```diff
@@ -55,14 +69,14 @@
                 }
             }
         }
-        .submit-button {
-            padding: 0 rem(15);
+        .form__button {
+            padding-left: rem(15);
```

Comment:
I think we want to leave 15px padding on the right so that this button is the same distance from the edge of the blue box as the Agency select.